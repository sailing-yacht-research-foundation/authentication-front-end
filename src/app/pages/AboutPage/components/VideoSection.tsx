import '../styles/video-section.css';
import React from 'react';
import styled from 'styled-components/macro';
import { Col, Row, Button, Space, Typography } from 'antd';
import { useHistory } from 'react-router';
import { useTranslation } from 'react-i18next';
import { translations } from 'locales/translations';
import SailingBackground from './../assets/hero-homepage-3.jpg';

export const VideoSection = (props) => {

    const { t } = useTranslation();

    const history = useHistory();

    return (
        <Wrapper className="video-section">
            <Row style={{ marginTop: '88px' }}>
                <Typography.Title className="text-white introduction-text">{t(translations.about_page.video_section.one_login)} <br /> {t(translations.about_page.video_section.ultimate_potential)}</Typography.Title>
            </Row>

            <Row style={{ marginTop: '40px', 'zIndex': 5 }}>
                <Space size={15}>
                    <Col>
                        <Button onClick={() => history.push('/signin')} className="syrf-button">{t(translations.about_page.video_section.login)}</Button>
                    </Col>

                    <Col>
                        <Button onClick={() => history.push('/signup')} className="syrf-button-outline">{t(translations.about_page.video_section.signup)}</Button>
                    </Col>
                </Space>
            </Row>
        </Wrapper>
    );
}

const Wrapper = styled.div`
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    align-items: center;
    min-height: 414px;
    width: 100%;
    background-size: cover;
    background-position: 50% 35%;
    text-align:center;
    padding: 0 15px;
    padding-bottom: 50px;
    position: relative;
    background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)), url(${SailingBackground});
`;