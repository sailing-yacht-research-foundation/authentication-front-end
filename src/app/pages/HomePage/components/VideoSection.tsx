import React from 'react';
import styled from 'styled-components/macro';
import { Col, Row, Button, Space, Typography } from 'antd';
import SailingBackground from '../assets/screen-0.jpg';
import { useHistory } from 'react-router';

export const VideoSection = (props) => {
    const history = useHistory();

    return (
        <Wrapper>
            <Row>
                <Typography.Title className="text-white uppercase">One Login</Typography.Title>
            </Row>
            <Row>
                <Typography.Title className="text-white uppercase">Ultimate Potential</Typography.Title>
            </Row>
            <Row >
                <Space size={15}>
                    <Col>
                        <Button onClick={() => history.push('/signin')} className="syrf-button-outline">Login</Button>
                    </Col>
                    <Col>
                        <Button onClick={() => history.push('/signup')} className="syrf-button">Signup</Button>
                    </Col>
                </Space>
            </Row>

        </Wrapper>
    );
}

const Wrapper = styled.div`
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    min-height: 550px;
    width: 100%;
    background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)), url(${SailingBackground});
    background-size: cover;
    margin-bottom: 70px;
    text-align:center;
`;

