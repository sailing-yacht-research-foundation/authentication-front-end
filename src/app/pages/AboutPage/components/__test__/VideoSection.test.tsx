import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { VideoSection } from '../VideoSection';

const shallowRenderer = createRenderer();

describe('<VideoSection />', () => {
  it('should render and matchs the snapshot', () => {
    shallowRenderer.render(<VideoSection />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});