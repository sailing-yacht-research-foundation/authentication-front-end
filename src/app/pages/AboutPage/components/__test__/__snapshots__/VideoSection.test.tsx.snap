// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoSection /> should render and matchs the snapshot 1`] = `
<VideoSection__Wrapper
  className="video-section"
>
  <Row
    style={
      Object {
        "marginTop": "88px",
      }
    }
  >
    <Title
      className="text-white introduction-text"
    >
      One Login
       
      <br />
       
      Ultimate Potential
    </Title>
  </Row>
  <Row
    style={
      Object {
        "marginTop": "40px",
        "zIndex": 5,
      }
    }
  >
    <Space
      size={15}
    >
      <Col>
        <Button
          className="syrf-button"
          onClick={[Function]}
        >
          Log In
        </Button>
      </Col>
      <Col>
        <Button
          className="syrf-button-outline"
          onClick={[Function]}
        >
          Sign Up
        </Button>
      </Col>
    </Space>
  </Row>
</VideoSection__Wrapper>
`;
